dvc/repo/ls.py

```diff
@@ -82,3 +130,26 @@ def _get_target_path_info(repo, target=None):
     if not target:
         return PathInfo(repo.root_dir)
     return PathInfo(repo.root_dir, target)
+
+
+def _get_fs_node(path_info, out=None):
```

Comment:
This one has two completely parallel code paths, may be split into two functions.

Reply:
I would like to keep as it is cause don't want produce additional functions with the same meaning. Also it will bloat the logic - needs dynamically choose right function, now it works inside one function

Reply:
Ok let's leave it as is.